validator/validation/validation.py

```diff
@@ -405,3 +432,31 @@ def stop_running_validation(validation_id):
     for task in celery_tasks:
         app.control.revoke(task.celery_task_id)  # @UndefinedVariable
         task.delete()
+
+
+def pytesmo_to_qa4sm_results(results):
+    # each key is a tuple of ((ds1, col1), (ds2, col2))
```

Comment:
Should this be a docstring? We have quite poor documentation in the `validation` module in general, but maybe it would be easier to develop when everything is better described

Reply:
This was actually more of an explanation for the next line, which is a bit hard to understand imo. I added a docstring nevertheless, good idea.